server/hpccloud/server/models/simulation.py

```diff
@@ -171,7 +171,7 @@ def update(self, user, simulation, name, metadata=None, description=None,
             dirty = True
             simulation['active'] = active
 
-        if disabled:
+        if disabled is not None:
```

Comment:
@cjh1 `disabled` is an array and `[]` is falsy. Does this check look ok instead?